templates/opencraft-nginx.j2

```diff
@@ -30,6 +30,17 @@ server {
     access_log /var/log/nginx/{{ OPENCRAFT_DOMAIN_NAME }}_access.log;
     error_log /var/log/nginx/{{ OPENCRAFT_DOMAIN_NAME }}_error.log error;
 
+    # serve static files
+    #
+    location /static {
+        alias {{ OPENCRAFT_STATIC_FILES_DIR }};
+        try_files $uri $uri/ =404;
+        autoindex off;
+        gzip on;
+        gzip_min_length 1000;
+        gzip_types text/css text/javascript application/javascript;
```

Comment:
Might be worth including `text/html` and `image/svg+xml` here.  (Angular templates are loaded as static assets, and they probably have MIME type `text/html`.)